tests/sentry/api/helpers/test_group_index.py

```diff
@@ -106,10 +106,7 @@ def test_unresolving_resolved_group(self, send_robust: Mock, send_unresolved: Mo
         request.data = {"status": "unresolved", "substatus": "ongoing"}
         request.GET = QueryDict(query_string=f"id={resolved_group.id}")
 
-        search_fn = Mock()
-        update_groups(
-            request, request.GET.getlist("id"), [self.project], self.organization.id, search_fn
-        )
+        update_groups(request, request.GET.getlist("id"), [self.project], self.organization.id)
```

Comment:
All the following changes are the same change. We don't test the `search_fn` in this file.